files/default/neutron-ha-tool.py

```diff
@@ -561,13 +563,14 @@ def replicate_dhcp(qclient, noop=False):
 
 
 def migrate_l3_routers_from_agent(qclient, agent, targets,
-                                  noop, wait_for_router, delete_namespace):
+                                  noop, wait_for_router, delete_namespace,
+                                  agent_picker):
```

Comment:
Ah sorry, I read too fast, this is indeed the only method which doesn't have optional arguments. I still find it a bit confusing (or at least, not so nice) that sometimes you have `agent_picker, noop` and in other cases `noop, ..., agent_picker`. But no big deal.